portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,16 @@
+package com.google.sps.servlets;
+
+public class Comment {
+  long id;
+  String text;
+  String timestamp;
```

Comment:
Why are you choosing a String to represent timestamp? That is a completely valid option, but probably you'll have to do more casts. Instead you can use a long. Usually that is a best to way to represent timestamp when we communicate between services.

Reply:
Fixed!